src/rgw/rgw_op.h

```diff
@@ -56,6 +56,7 @@
 #include "rgw_public_access.h"
 #include "rgw_bucket_encryption.h"
 #include "rgw_tracer.h"
+#include "rgw_multi_del.h"
```

Comment:
please just forward declare `class RGWMultiDelObject` instead of including the header